values-schema.yaml

```diff
@@ -430,42 +423,33 @@ definitions:
             enum:
               - docker-registry
             type: string
-          secretPath:
-            $ref: '#/definitions/vaultPath'
-          itemName:
+          dockerconfig:
             type: string
-            default: '.dockerconfig.json'
+            enum:
+              - '.dockerconfig.json'
             readOnly: true
         required:
           - type
-          - secretPath
       - title: TLS
         properties:
           type:
             default: tls
             enum:
               - tls
             type: string
-          crtSecretPath:
-            $ref: '#/definitions/vaultPath'
-          crtItemName:
+          crt:
             type: string
-            default: 'crt'
-          keySecretPath:
-            $ref: '#/definitions/vaultPath'
-          keyItemName:
+            default: 'tls.crt'
+          key:
             type: string
-            default: 'key'
-          caSecretPath:
-            $ref: '#/definitions/vaultPath'
-          caItemName:
+            default: 'tls.key'
+          ca:
             type: string
-            default: 'ca'
+            default: 'ca.crt'
         required:
           - type
-          - caSecretPath
-          - crtSecretPath
-          - keySecretPath
+          - crt
+          - key
```

Comment:
We can't reuse the `secret` definition? Ie.

```yaml
secret:
    oneOf:
      - properties:
          password:
            type: string
          server:
            $ref: '#/definitions/registry'
          type:
            default: docker-registry
            enum:
              - docker-registry
            type: string
          username:
            type: string
        required:
          - server
          - username
          - password
        title: Docker Registry
      - properties:
          entries:
            description: Key names and their values.
            items:
              properties:
                key:
                  description: Key or file name.
                  example: secretstuff.conf
                  type: string
                value:
                  description: Value to be encrypted.
                  example: password=yoursecretpassword
                  type: string
              required:
                - key
                - value
            type: array
          type:
            default: generic
            enum:
              - generic
            type: string
        required:
          - entries
        title: Generic
      - properties:
          ca:
            description: The CA certificate content.
            type: string
          crt:
            description: The PEM encoded public key certificate content.
            type: string
          key:
            description: The private key certificate content.
            type: string
          type:
            default: tls
            enum:
              - tls
            type: string
        required:
          - crt
          - key
        title: TLS
```

It looks very similar to me... I could be wrong of course, e.g. if this is the way to use them simultaneously in the transition?

Reply:
This is a new schema is required to work with Vault.
It this is a question then please to not click 'changes requested' :P

Reply:
No we can't. Because Vault stores 

> Hmmm...too many reviews. I still see demo values using old list. No changes there yet

where ?

Reply:
> This is a new schema is required to work with Vault.

Yes, I just don't want to give the impression and act like I know it all... 

So we can't extend the original Secret schema, since they are a subset of one another?

Reply:
> where ? I do not see old list. We did not agree to use named properties.

_I think this PR is easier to read if you reply to @Morriz in a reply to the issue and not to my review..._

Reply:
> where ? I do not see old list. We did not agree to use named properties.

Maybe you missed our little chat here in this PR: https://github.com/redkubes/otomi-core/pull/360/files#r594533327